### ISSUE TITLE
Accessibility: Find a replacement for presentational-only CSS `content`

### ISSUE THREAD

**Issue by horazont** [snikket-im/snikket-web-portal#41]
Problem: Some screenreaders read that.

Potential fixes:

- `content: "presentational" / "alt"`. Doesn’t work except in chrome, and in other browsers this causes the "alt" to be rendered, which we’d want to be blank, thus breaking the layout for non-screenreader use.
- `@media speech`: Does not seem to work with the affected devices.
- Replace the content with SVG images (preferred), is a bit of work though.
